core/pta/device.c

```diff
@@ -82,18 +86,31 @@ static TEE_Result invoke_command(void *pSessionContext __unused,
 				 uint32_t nCommandID, uint32_t nParamTypes,
 				 TEE_Param pParams[TEE_NUM_PARAMS])
 {
+	TEE_Result res = TEE_SUCCESS;
+	uint32_t rflags = 0;
+
 	switch (nCommandID) {
 	case PTA_CMD_GET_DEVICES:
-		return get_devices(nParamTypes, pParams,
-				   TA_FLAG_DEVICE_ENUM);
+		rflags = TA_FLAG_DEVICE_ENUM;
+		break;
 	case PTA_CMD_GET_DEVICES_SUPP:
-		return get_devices(nParamTypes, pParams,
-				   TA_FLAG_DEVICE_ENUM_SUPP);
-	default:
+		rflags = TA_FLAG_DEVICE_ENUM_SUPP;
+		if (IS_ENABLED(CFG_REE_FS))
+			rflags |= TA_FLAG_DEVICE_ENUM_SEC_STORAGE;
 		break;
+	case PTA_CMD_GET_DEVICES_RPMB:
+		if (!IS_ENABLED(CFG_REE_FS)) {
```

Comment:
`IS_ENABLED(CFG_RPMB_FS)`, no?

Reply:
No, this is supposed to be the complement of `IS_ENABLED(CFG_REE_FS)` above.
RPMB only needs to be initialized if the `TA_FLAG_DEVICE_ENUM_SEC_STORAGE` should be used here.